app/views/visns/_show_table_row.html.erb

```diff
@@ -1,10 +1,30 @@
 <% facilities.each do |facility| %>
+  <% path = facility.class.name === "ClinicalResourceHub" ? "/crh/" + facility.visn.number.to_s : va_facility_path(facility)  %>
```

Comment:
I think you can do `clinical_resource_hub_path(facility)` here

Reply:
Yep good catch.